homeassistant/components/fyta/sensor.py

```diff
@@ -0,0 +1,171 @@
+"""Summary data from Fyta."""
+from __future__ import annotations
+
+from collections.abc import Callable
+from dataclasses import dataclass
+from datetime import datetime
+from typing import Final
+
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorEntityDescription,
+    SensorStateClass,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import PERCENTAGE, EntityCategory
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+from .coordinator import FytaCoordinator, FytaEntity
+
+
+@dataclass(frozen=True)
+class FytaSensorEntityDescription(SensorEntityDescription):
+    """Describes Fyta sensor entity."""
+
+    value_fn: Callable[[str | int | float], str | int | float | datetime] = (
+        lambda value: value
+    )
+
+
+SENSORS: Final[list[FytaSensorEntityDescription]] = [
+    FytaSensorEntityDescription(
+        key="plant_name",
+        translation_key="fyta_plant_name",
+    ),
+    FytaSensorEntityDescription(
+        key="scientific_name",
+        translation_key="fyta_scientific_name",
+    ),
+    FytaSensorEntityDescription(
+        key="plant_status",
+        translation_key="fyta_plant_status",
+        state_class=SensorStateClass.MEASUREMENT,
+        icon="mdi:flower",
+    ),
+    FytaSensorEntityDescription(
+        key="temperature_status",
+        translation_key="fyta_temperature_status",
+        state_class=SensorStateClass.MEASUREMENT,
+        icon="mdi:thermometer-lines",
+    ),
+    FytaSensorEntityDescription(
+        key="light_status",
+        translation_key="fyta_light_status",
+        state_class=SensorStateClass.MEASUREMENT,
+        icon="mdi:sun-clock-outline",
+    ),
+    FytaSensorEntityDescription(
+        key="moisture_status",
+        translation_key="fyta_moisture_status",
+        state_class=SensorStateClass.MEASUREMENT,
+        icon="mdi:water-percent-alert",
+    ),
+    FytaSensorEntityDescription(
+        key="salinity_status",
+        translation_key="fyta_salinity_status",
+        state_class=SensorStateClass.MEASUREMENT,
+        icon="mdi:sprout-outline",
+    ),
+    FytaSensorEntityDescription(
+        key="temperature",
+        translation_key="fyta_temperature",
+        native_unit_of_measurement="°C",
```

Comment:
Please use constants for this


```suggestion
        native_unit_of_measurement=UnitOfTemperature.CELCIUS,
```